docs/user/installation.rst

```diff
@@ -76,10 +57,48 @@ Alternatively, you can add garage in the pip section of your `environment.yml`
     pip install garage
 
 
+Install System Dependencies (Optional)
+--------------------------------------
+
+Generally speaking, system dependencies of garage are minimal, and likely already installed.
+However, many of the environments garage is used with have additional
+dependencies, and we provide "setup scripts" for installing those dependencies
+and working around known problems on common platforms.
+
+If you can already use the environments you need, you can skip this section.
+
+A MuJoCo key is required to run these install scripts. You can get one here: https://www.roboti.us/license.html
+
+In order to use those scripts, please do the following:
+
+Clone our repository (https://github.com/rlworkgroup/garage) and navigate to its directory.
+
+Then, from the root directory of the repo, run the script.
+
+- On Linux:
+
+.. code-block:: bash
+
+    ./scripts/setup_linux.sh --mjkey path-to-your-mjkey.txt --modify-bashrc
+
+- On macOS:
+
+.. code-block:: bash
+
+    ./scripts/setup_macos.sh --mjkey path-to-your-mjkey.txt --modify-bashrc
```

Comment:
The --modify-bashrc argument is invalid in setup_macox.sh. This should be
` ./scripts/setup_macos.sh --mjkey path-to-your-mjkey.txt --modify-shellrc`

Reply:
Good catch, thanks!